src-docs/src/components/guide_page/guide_page_header.tsx

```diff
@@ -62,6 +62,25 @@ export const GuidePageHeader: React.FunctionComponent<{}> = () => {
     );
   }
 
+  function renderFigma() {
+    const href = 'https://www.figma.com/community/file/1240039515407642648';
+    const label = 'OUI Figma component library';
+    return isMobileSize ? (
+      <OuiButtonEmpty size="s" flush="both" iconType="logoFigma" href={href}>
+        {label}
+      </OuiButtonEmpty>
+    ) : (
+      <OuiToolTip content="Figma Component Library">
+        <OuiHeaderSectionItemButton
+          aria-label={label}
+          // @ts-ignore TODO: FIX
```

Comment:
what's the typescript error here?

Reply:
I actually don't know, I copy-pasted the renderGithub() and modified based on that. Now that I have looked at it, I think it is related to prettier formatting, already fixed it.